docs/external/ccp.md

```
@@ -0,0 +1,49 @@
+# CCP Platform
+
+The CCP Platform is used exclusively to rotate the OpenID client secret that expires on a monthly basis. The extension calls the CCP API to retrieve a fresh `clientSecret`, then stores it in Azure Key Vault for use by the `OpenIDClient`.
+
+## Authentication
+
+OAuth 2.0 Client Credentials. The extension authenticates against the CCP OAuth endpoint using `EXT_CCP_OAUTH_CREDENTIALS_SCOPE` (a different scope than the one used for Cloud Orchestrator).
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)